### PR TITLE
ticdc: Update cdc server config

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-server-config.md [hunk 1]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
   cert-path = ""
   key-path = ""
 
-
+# TiCDC 与 etcd 服务间的 session 时长(单位为秒)，默认为 10，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:49]:
```suggestion
# TiCDC 与 etcd 服务间的 session 时长（单位为秒），默认为 10，可选。 
```

File: ticdc/ticdc-server-config.md [hunk 2]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
   cert-path = ""
   key-path = ""
 
-
+# TiCDC 与 etcd 服务间的 session 时长(单位为秒)，默认为 10，可选。 
 capture-session-ttl = 10 # 10s
+# TiCDC 集群中的 owner 模块尝试推进同步任务进度的周期，默认为 "50ms",  可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:51]:
```suggestion
# TiCDC 集群中的 owner 模块尝试推进同步任务进度的周期，默认为 "50ms"，可选。
```

File: ticdc/ticdc-server-config.md [hunk 3]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
 owner-flush-interval = 50000000 # 50 ms
+# TiCDC 集群中的 processor 模块尝试推进同步任务进度的周期，默认为 "50ms",  可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:53]:
```suggestion
# TiCDC 集群中的 processor 模块尝试推进同步任务进度的周期，默认为 "50ms"，可选。  
```

File: ticdc/ticdc-server-config.md [hunk 4]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
-per-table-memory-quota = 10485760 # 10 MiB
 
-[log]
-  error-output = "stderr"
-  [log.file]
-    max-size = 300 # 300 MiB
-    max-days = 0
-    max-backups = 0
+#[log]
+#  用于指定 zap log 模块的内部错误日志输出到哪里。  默认是 "stderr"，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:57]:
```suggestion
#  用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。  
```

File: ticdc/ticdc-server-config.md [hunk 5]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
-  error-output = "stderr"
-  [log.file]
-    max-size = 300 # 300 MiB
-    max-days = 0
-    max-backups = 0
+#[log]
+#  用于指定 zap log 模块的内部错误日志输出到哪里。  默认是 "stderr"，可选。  
+#  error-output = "stderr"
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为（MB) 。默认是 300， 可选
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:60]:
```suggestion
#    单个 log 文件的最大文件大小，单位为 MB。默认是 300， 可选
```

File: ticdc/ticdc-server-config.md [hunk 6]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#  Sorter 文件 相对于 data-dir 的目录，默认值 "/tmp/sorter", 可选  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 region worker 中可使用的处理线程数，默认为 8， 可选。  
 #   worker-concurrent = 8
+#   TiCDC 中共享线程池的大小，主要用于处理 kv 事件，默认为 0， 表示为 cpu 核数的 2 倍， 可选。   
 #   worker-pool-size = 0
-#   region-scan-limit = 40
+#   Region 连接重试时间， 默认值为 1 分钟， 可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:78]:
```suggestion
#   Region 连接重试时间，默认值为 1 分钟，可选。
```

File: ticdc/ticdc-server-config.md [hunk 7]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MB, 默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件 相对于 data-dir 的目录，默认值 "/tmp/sorter", 可选  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 region worker 中可使用的处理线程数，默认为 8， 可选。  
 #   worker-concurrent = 8
+#   TiCDC 中共享线程池的大小，主要用于处理 kv 事件，默认为 0， 表示为 cpu 核数的 2 倍， 可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:76]:
```suggestion
#   TiCDC 中共享线程池的大小，主要用于处理 kv 事件，默认为 0， 表示为 cpu 核数的 2 倍， 可选。   
```
```suggestion
#   TiCDC 中共享线程池的大小，主要用于处理 KV 事件，默认为 0， 表示为 CPU 核数的 2 倍， 可选。   
```

Comment (qiancai) [reply ticdc/ticdc-server-config.md:76]: 那 1 表示什么呢

File: ticdc/ticdc-server-config.md [hunk 8]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#    max-size = 300 # 300 MiB
+#    log 文件最大保留天数，默认是永不删除（0），可选。  
+#    max-days = 0
+#    保留 log 文件数， 默认是保留所有(0)， 可选  
+#    max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MB, 默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件 相对于 data-dir 的目录，默认值 "/tmp/sorter", 可选
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:70]:
```suggestion
#  Sorter 文件相对于 data-dir 的目录，默认值 "/tmp/sorter"，可选  
```

File: ticdc/ticdc-server-config.md (hunk 7 above)
Comment (sdojjy) [reply ticdc/ticdc-server-config.md:76]: 表示线程池里只有一个线程

File: ticdc/ticdc-server-config.md [hunk 9]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
-  error-output = "stderr"
-  [log.file]
-    max-size = 300 # 300 MiB
-    max-days = 0
-    max-backups = 0
+#[log]
+#  用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。  
+#  error-output = "stderr"
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MB。默认是 300， 可选
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:60]:
```suggestion
#    单个 log 文件的最大文件大小，单位为 MB。默认是 300，可选。
```

File: ticdc/ticdc-server-config.md [hunk 10]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#    max-size = 300 # 300 MiB
+#    log 文件最大保留天数，默认是永不删除（0），可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:62]:
```suggestion
#    log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。  
```

File: ticdc/ticdc-server-config.md [hunk 11]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
-    max-backups = 0
+#[log]
+#  用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。  
+#  error-output = "stderr"
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MB。默认是 300， 可选
+#    max-size = 300 # 300 MiB
+#    log 文件最大保留天数，默认是永不删除（0），可选。  
+#    max-days = 0
+#    保留 log 文件数， 默认是保留所有(0)， 可选
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:64]:
```suggestion
#    log 文件数的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。  
```

File: ticdc/ticdc-server-config.md [hunk 12]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MB。默认是 300， 可选
+#    max-size = 300 # 300 MiB
+#    log 文件最大保留天数，默认是永不删除（0），可选。  
+#    max-days = 0
+#    保留 log 文件数， 默认是保留所有(0)， 可选  
+#    max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MB, 默认值为 128.
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:68]:
```suggestion
#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128.  
```

File: ticdc/ticdc-server-config.md [hunk 13]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#  Sorter 文件相对于 data-dir 的目录，默认值 "/tmp/sorter"，可选  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 region worker 中可使用的处理线程数，默认为 8， 可选。  
 #   worker-concurrent = 8
+#   TiCDC 中共享线程池的大小，主要用于处理 KV 事件，默认为 0， 表示为 CPU 核数的 2 倍， 可选。   
 #   worker-pool-size = 0
-#   region-scan-limit = 40
+#   Region 连接重试时间，默认值为 1 分钟，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:78]: 单位是什么呢

File: ticdc/ticdc-server-config.md [hunk 14]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MB, 默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件相对于 data-dir 的目录，默认值 "/tmp/sorter"，可选  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 region worker 中可使用的处理线程数，默认为 8， 可选。  
 #   worker-concurrent = 8
+#   TiCDC 中共享线程池的大小，主要用于处理 KV 事件，默认为 0， 表示为 CPU 核数的 2 倍， 可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:76]:
```suggestion
#   TiCDC 中共享线程池中线程的数量，主要用于处理 KV 事件，默认值为 `0`，表示CPU 核数的 2 倍，可选。   
```

File: ticdc/ticdc-server-config.md [hunk 15]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#    max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MB, 默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件相对于 data-dir 的目录，默认值 "/tmp/sorter"，可选  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 region worker 中可使用的处理线程数，默认为 8， 可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:74]:
```suggestion
#   单个 Region worker 中可使用的线程数量，默认为 8，可选。  
```

Comment (qiancai) [reply ticdc/ticdc-server-config.md:74]: "concurrent" 是代表 “可并行使用的线程数量“ 吗？

File: ticdc/ticdc-server-config.md [hunk 16]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#    max-size = 300 # 300 MiB
+#    log 文件最大保留天数，默认是永不删除（0），可选。  
+#    max-days = 0
+#    保留 log 文件数， 默认是保留所有(0)， 可选  
+#    max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MB, 默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件相对于 data-dir 的目录，默认值 "/tmp/sorter"，可选
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:70]:
```suggestion
#  Sorter 文件相对于 data-dir 的目录，默认值为 "/tmp/sorter"，可选。  
```

File: ticdc/ticdc-server-config.md (hunk 13 above)
Comment (sdojjy) [reply ticdc/ticdc-server-config.md:78]: 可以加单位，比如 "60s" 表示 60分钟

Comment (qiancai) [reply ticdc/ticdc-server-config.md:78]: 不加单位的话，默认是什么单位呢

File: ticdc/ticdc-server-config.md (hunk 9 above)
Comment (qiancai) [inline ticdc/ticdc-server-config.md]:
```suggestion
#    单个 log 文件的最大文件大小，单位为 MB。默认是 300，可选
```

File: ticdc/ticdc-server-config.md (hunk 13 above)
Comment (sdojjy) [reply ticdc/ticdc-server-config.md:78]: 纳秒

Comment (qiancai) [reply ticdc/ticdc-server-config.md:78]: 也可以设置为  60ms 吗

File: ticdc/ticdc-server-config.md [hunk 17]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#  Sorter 文件相对于 data-dir 的目录，默认值为 "/tmp/sorter"，可选。  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 Region worker 中可使用的线程数量，默认为 8，可选。  
 #   worker-concurrent = 8
+#   TiCDC 中共享线程池中线程的数量，主要用于处理 KV 事件，默认值为 `0`，表示CPU 核数的 2 倍，可选。   
 #   worker-pool-size = 0
-#   region-scan-limit = 40
+#   Region 连接重试时间，默认值为 1 分钟，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:78]:
```suggestion
#   Region 连接重试时间，默认值为 `60000000000` 纳秒，即 1 分钟，可选。该参数有两种配置方式：只指定数字（例如，配置为 `50000000` 表示 50000000 纳秒，即 50 毫秒），或同时指定数字和单位（例如，直接配置为 `50ms`）。
```

File: ticdc/ticdc-server-config.md [hunk 18]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#    log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。  
+#    max-days = 0
+#    log 文件数的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。  
+#    max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件相对于 data-dir 的目录，默认值为 "/tmp/sorter"，可选。  
+#  sorter-dir = ""
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:71]:
```suggestion
#  sorter-dir = "/tmp/sorter"
```

File: ticdc/ticdc-server-config.md [hunk 19]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MB。默认是 300，可选。
+#    max-size = 300 # 300 MiB
+#    log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。  
+#    max-days = 0
+#    log 文件数的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。  
+#    max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128.
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:68]:
```suggestion
#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128。 
```

File: ticdc/ticdc-server-config.md [hunk 20]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
-    max-backups = 0
+#[log]
+#  用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。  
+#  error-output = "stderr"
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MB。默认是 300，可选。
+#    max-size = 300 # 300 MiB
+#    log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。  
+#    max-days = 0
+#    log 文件数的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:64]:
```suggestion
#    log 文件的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。  
```

File: ticdc/ticdc-server-config.md [hunk 21]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
-  error-output = "stderr"
-  [log.file]
-    max-size = 300 # 300 MiB
-    max-days = 0
-    max-backups = 0
+#[log]
+#  用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。  
+#  error-output = "stderr"
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MB。默认是 300，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:60]:
```suggestion
#    单个 log 文件的最大文件大小，单位为 MiB。默认值为 300，可选。
```

File: ticdc/ticdc-server-config.md [hunk 22]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
   cert-path = ""
   key-path = ""
 
-
+# TiCDC 与 etcd 服务间的 session 时长（单位为秒），默认为 10，可选。 
 capture-session-ttl = 10 # 10s
+# TiCDC 集群中的 owner 模块尝试推进同步任务进度的周期，默认为 "50ms"，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:51]:
```suggestion
# TiCDC 集群中的 owner 模块尝试推进同步任务进度的周期，默认值为 `50000000` 纳秒（即 50 毫秒），可选。该参数有两种配置方式：只指定数字（例如，配置为 `40000000` 表示 40000000 纳秒，即 40 毫秒），或同时指定数字和单位（例如，直接配置为 `40ms`）。
```

File: ticdc/ticdc-server-config.md [hunk 23]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
 owner-flush-interval = 50000000 # 50 ms
+# TiCDC 集群中的 processor 模块尝试推进同步任务进度的周期，默认为 "50ms"，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:53]:
```suggestion
# TiCDC 集群中的 processor 模块尝试推进同步任务进度的周期，默认值为 `50000000` 纳秒（即 50 毫秒），可选。该参数配置方式与 `owner-flush-interval` 相同。
```

File: ticdc/ticdc-server-config.md [hunk 24]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128.  
+#  cache-size-in-mb = 128
+#  Sorter 文件相对于 data-dir 的目录，默认值为 "/tmp/sorter"，可选。  
+#  sorter-dir = ""
 
 # [kv-client]
+#   单个 Region worker 中可使用的线程数量，默认为 8，可选。  
 #   worker-concurrent = 8
+#   TiCDC 中共享线程池中线程的数量，主要用于处理 KV 事件，默认值为 `0`，表示CPU 核数的 2 倍，可选。
```
Comment (qiancai) [inline ticdc/ticdc-server-config.md:76]:
```suggestion
#   TiCDC 中共享线程池中线程的数量，主要用于处理 KV 事件，默认值为 `0`，表示 CPU 核数的 2 倍，可选。   
```

Comment (qiancai) [reply ticdc/ticdc-server-config.md:76]: 如果设置为 1，代表 表示 CPU 核数的 4 倍 吗？

Comment (sdojjy) [reply ticdc/ticdc-server-config.md:76]: 不是， 0 表示没有设置，系统会选一个默认值

File: ticdc/ticdc-server-config.md [hunk 25]
```diff
@@ -31,6 +31,7 @@ summary: 了解 TiCDC 详细的命令行参数和配置文件定义。
 对于 `cdc server` 命令中 config 参数指定的配置文件说明如下：
```
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:31]:
```suggestion
对于 `cdc server` 命令中 `config` 参数指定的配置文件说明如下：
```

File: ticdc/ticdc-server-config.md [hunk 26]
```diff
@@ -31,6 +31,7 @@ summary: 了解 TiCDC 详细的命令行参数和配置文件定义。
...
 
 ```yaml
```
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:33]:
```suggestion
```toml
```

File: ticdc/ticdc-server-config.md [hunk 27]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#[log]
+#  用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。  
+#  error-output = "stderr"
+#  [log.file]
+#    单个 log 文件的最大文件大小，单位为 MiB。默认值为 300，可选。
+#    max-size = 300 # 300 MiB
+#    log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。  
+#    max-days = 0
+#    log 文件的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。  
+#    max-backups = 0
```
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:65]:
```suggestion
# [log]
# # 用于指定 zap log 模块内部的错误日志的输出位置。默认是 "stderr"，可选。
#   error-output = "stderr"
#   [log.file]
#     # 单个 log 文件的最大文件大小，单位为 MiB。默认值为 300，可选。
#     max-size = 300 # 300 MiB
#     # log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。
#     max-days = 0
#     # log 文件的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。
#     max-backups = 0
```
建议修改为这种格式，这样用户对整个 section 取消注释后，依然是有效的语法格式、一致的缩进

**Review by qiancai — APPROVED** [review]

File: ticdc/ticdc-server-config.md [hunk 28]
```diff
@@ -45,23 +46,35 @@ cluster-id = "default"
...
+#     # log 文件最长保留天数，默认值为 `0`，代表永不删除，可选。
+#     max-days = 0
+#     # log 文件的保留个数，默认值为 `0`，代表保留所有 log 文件，可选。
+#     max-backups = 0
 
+#[sorter]
+#  Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128。 
+#  cache-size-in-mb = 128
+#  Sorter 文件相对于 data-dir 的目录，默认值为 "/tmp/sorter"，可选。  
+#  sorter-dir = "/tmp/sorter"
```
Comment (Oreoxmt) [inline ticdc/ticdc-server-config.md:71]:
```suggestion
# [sorter]
# # Sorter 模块给默认启动的 8 个 pebble DB 共享的 pebble block cache 的大小，单位为 MiB，默认值为 128。
#   cache-size-in-mb = 128
#   # Sorter 文件相对于 data-dir 的目录，默认值为 "/tmp/sorter"，可选。
#   sorter-dir = "/tmp/sorter"
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**